ember/app/controllers/index.js

```diff
@@ -21,6 +21,14 @@ export default Ember.Controller.extend({
   time: '17:30',
   date: new Date().toISOString().substring(0, 10),
 
+  popRes: function(){
+	 console.log(this.get('model.user.id'));
+      this.get('ajax')
+          .request('/getPopularRestaurants/' + this.get('model.user.id'))
+          .then(result => this.set('popRes', result));
```

Comment:
Never send any requests from a controller. There are services for this. Also, this should be loaded with the model when the page loads, in the route.